source/overview/product.rst

```diff
@@ -2,70 +2,154 @@
 Product Overview
 ============================
 
-**Mattermost** empowers organizations to achieve their highest priorities through modern, enterprise-grade communication. Enjoy all the productivity benefits of workplace messaging across web, mobile and PC, with unlimited archiving, search and integrations in a single-tenant, private cloud solution under IT control.
+Mattermost allows organizations to achieve their highest goals through enterprise-grade communication, 
+and to enjoy all the productivity benefits of workplace messaging across web, 
+mobile, 
+and PC.
+This is coupled with unlimited archiving, 
+searching,
+and integrating in a single-tenant, 
+private cloud solution under IT control.
 
-Thousands of organizations use Mattermost around the world in 14 languages for its unmatched benefits:
+Mattermost is used in 14 languages by thousands of organizations around the world and is valued for its unmatched benefits in security,
+configurability,
+and scalability.
 
-- **Security** - Keep vital communications, including access to mobile and desktop apps, behind your firewall. Deploy using `dozens of security features <https://docs.mattermost.com/overview/security.html>`_ vetted by top security researchers. Data stays on servers you control, encrypted using keys you control.
+Security
+~~~~~~~~-
 
-- **Configurability** - Adapt your deployment to your needs, preferences, and existing systems. The open source Mattermost server enables advanced white labelling and customization with complete access to RESTful server APIs, drivers, webhooks and hundreds of 3rd party extensions.
+Mattermost keeps vital communications like access to mobile and desktop apps behind your firewall and can be deployed using `dozens of security features <https://docs.mattermost.com/overview/security.html>`_ vetted by top security researchers. 
+Data stays on servers you control and is encrypted using keys you control.
 
-- **Scalability** - Grow from dozens of users to tens of thousands on the same server. Built on a high performance, single-tenant infrastructure, Mattermost E20 offers cluster-based high availability deployments with horizontal scaling and advanced performance monitoring.
+Configurability 
+~~~~~~~~~~~~~~-
 
-About the Mattermost open source project
+Mattermost allows organizations to adapt deployments to their needs, 
+preferences, 
+and existing systems. 
+The open source Mattermost server enables advanced white labelling and customization with complete access to RESTful server APIs, 
+drivers, 
+webhooks, 
+and hundreds of third party extensions.
+
+Scalability 
+~~~~~~~~~~~~
+
+Mattermost can scale from dozens to tens of thousands of users on the same server. 
+The Mattermost E20 offers cluster-based high availability deployments with horizontal scaling and advanced performance monitoring atop single-tenant infrastructure with high performance.
```

Comment:
"Mattermost E20" or "The Mattermost E20 edition"

Reply:
it varies depending on the context....